### PR TITLE
Add Clementine.app v1.3rc1

### PR DESCRIPTION
**Pull request by sanyer**

### CODE REVIEW

File: Casks/clementine-rc.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  appcast 'https://github.com/clementine-player/Clementine/releases.atom',
+          checkpoint: '460ea33daaedccd5c58ce4cda1ace8c86827ba3533dac9a5d0ca0df5e1e1f875'
+  name 'Clementine'
+  homepage 'https://www.clementine-player.org/'
+  license :gpl
+
+  conflicts_with cask: 'caskroom/clementine'
+
+  app 'Clementine.app'
+  binary 'Clementine.app/Contents/MacOS/clementine'
```
Comment (vitorgalvao) [inline Casks/clementine-rc.rb]: This isn’t an appropriate use of `binary`. Please remove it.

File: Casks/clementine-rc.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  # github.com/clementine-player/Clementine was verified as official when first introduced to the cask
+  url "https://github.com/clementine-player/Clementine/releases/download/#{version.gsub(%r{.0}, '')}/clementine-#{version}.dmg"
+  appcast 'https://github.com/clementine-player/Clementine/releases.atom',
+          checkpoint: '460ea33daaedccd5c58ce4cda1ace8c86827ba3533dac9a5d0ca0df5e1e1f875'
+  name 'Clementine'
+  homepage 'https://www.clementine-player.org/'
+  license :gpl
+
+  conflicts_with cask: 'caskroom/clementine'
```
Comment (vitorgalvao) [inline Casks/clementine-rc.rb]: Tis won’t work. There’s no such repo. Needs to be `'caskroom/cask/clementine'`.

File: Casks/clementine-rc.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  app 'Clementine.app'
+  binary 'Clementine.app/Contents/MacOS/clementine'
+
+  uninstall quit: 'org.clementine-player.Clementine'
```
Comment (vitorgalvao) [inline Casks/clementine-rc.rb]: Not needed.

File: Casks/clementine-rc.rb (hunk 2 above)
Comment (vitorgalvao) [inline Casks/clementine-rc.rb]: Please port this over as `conflicts_with cask: 'caskroom/versions/clementine-rc'` to the main cask.

File: Casks/clementine-rc.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+  zap delete: [
```
Comment (vitorgalvao) [inline Casks/clementine-rc.rb]: Mind porting this to the main cask?

Comment (sanyer) [inline Casks/clementine-rc.rb]: @vitorgalvao I think it's reasonable, previous 1.2.2 version is quite old. This one is not actually a beta.

File: Casks/clementine-rc.rb (hunk 3 above)
Comment (sanyer) [inline Casks/clementine-rc.rb]: It was night, I was sleeping 10 hours ago :-)

**Comment by sanyer** [comment]
done
[pull request to main cask](https://github.com/caskroom/homebrew-cask/pull/20269)

File: Casks/clementine-rc.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
+cask 'clementine-rc' do
+  version '1.3.0rc1'
+  sha256 'dd578a53c7fd47c89ade55eae224b5390590a32de982479ecf73e738d6620249'
+
+  # github.com/clementine-player/Clementine was verified as official when first introduced to the cask
+  url "https://github.com/clementine-player/Clementine/releases/download/#{version.gsub(%r{.0}, '')}/clementine-#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/clementine-rc.rb]: No need to use a regex, there.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 0df50018f046637b2d8d39051dc6ee347a85cb93. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/clementine-rc.rb (hunk 5 above)
Comment (sanyer) [inline Casks/clementine-rc.rb]: agreed, regex is not a need

**Comment by sanyer** [comment]
So, were there any other fixes except getting rid of regex?

**Comment by vitorgalvao** [comment]
Nope.

**Comment by sanyer** [comment]
ok, cool :smile:

**Pull request closed without merging**